.github/workflows/ci.yml

```diff
@@ -1,75 +1,95 @@
-name: Full Service CI
+name: ci
 
 env:
-  DOCKERHUB_REPO: mobilecoin/full-service
+  SGX_MODE: HW
+  IAS_MODE: PROD
+  RUST_BACKTRACE: full
+  CONSENSUS_ENCLAVE_CSS: /var/tmp/consensus-enclave.css
+  INGEST_ENCLAVE_CSS: /var/tmp/ingest-enclave.css
 
 on:
-  push:
-    tags:
-      - 'v*'
+  pull_request:
     branches:
-      - 'develop'
-      - 'feature/**'
+      - develop
+      - main
 
 jobs:
-  # Stub out job for work Mikey is working on for stress tests
-  # tests:
-  #   runs-on: ubuntu-latest
-  #   steps:
-  #     - name: CI Tests
-  #       run: |
-  #         echo "Hello World"
+  lint:
+    runs-on: [self-hosted, Linux, large]
+    container:
+      image: mobilecoin/rust-sgx-base:0.0.4
```

Comment:
Optional/Equivalent: Use `:latest` (throughout this PR) to get the latest image without having to update this YAML.